test/test_xla_dist.py

```diff
@@ -704,6 +721,121 @@ def test_healthy_cluster(self):
         client_master_ip='10.0.0.0')
     self.assertEqual(expected, cluster)
 
+  def test_healthy_remote_coordinator(self):
+    noop_compute_service = build_mock_compute_service({}, {})
+    self.mock_discovery.side_effect = build_mock_services_fn(
+        noop_compute_service)
+
+    tpu_map = {
+        'fake-pod': {
+            'state':
+                'READY',
+            'health':
+                'HEALTHY',
+            'runtime_version':
+                'v2-nightly',
```

Comment:
This reminds me, let's add on our user guide that users must not use `v2-.*-pod` TPU runtime versions as that one also starts TF server up and our XRT server will get "/dev/accel[0-4]  already in use" errors.

Reply:
yea, I will mention that.